src/main.cpp

```diff
@@ -3084,13 +3084,15 @@ bool ContextualCheckBlock(const CBlock& block, CValidationState& state, CBlockIn
     }
 
     // Coinbase transaction must include an output sending 20% of
-    // the block reward to `FOUNDERS_REWARD_SCRIPT` until the first
-    // subsidy halving block, with exception to the genesis block.
-    if ((nHeight > 0) && (nHeight < consensusParams.nSubsidyHalvingInterval)) {
+    // the block reward to a founders reward script, until the last founders
+    // reward block is reached, with exception of the genesis block.
+    // The last founders reward block is defined as the block just before
+    // the first subsidy halving block plus the subsidy slow start shift.
```

Comment:
Not quite: it is defined as the block just before the first subsidy halving block, which occurs at `halving_interval + slow_start_shift`.